src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -64,6 +64,10 @@ a#editUncommonSettingsButton:hover{
     padding-top: 10px;
 }
 
+.bold-label {
+    font-weight: 700 !important;
```

Comment:
This was just to preserve the original style of the page where labels had `font-weight: 700`. Making it responsive caused the default bootstrap label style to get applied.